Dockerfile

```diff
@@ -0,0 +1,63 @@
+# For more information, please refer to https://aka.ms/vscode-docker-python
+FROM python:3.8-slim-buster as python-base
+
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE=1 \
+# Turns off buffering for easier container logging
+PYTHONUNBUFFERED=1 \
+POETRY_VERSION=1.0.0 \
+ # make poetry install to this location
+POETRY_HOME=".poetry/bin"
+# paths
+# this is where our requirements + virtual environment will live
+#PYSETUP_PATH="/opt/pysetup"
+
+# `builder-base` stage is used to build deps 
+FROM python-base as builder-base
```

Comment:
This seems unnecessary. You could have done the builder-base steps in python-base